### PR TITLE
automatically try install kvm for kvm2 tests

### PR DESCRIPTION
**Pull request by sharifelgamal**

### CODE REVIEW

File: hack/jenkins/linux_integration_tests_kvm.sh [hunk 1]
```diff
@@ -36,4 +36,8 @@ EXTRA_TEST_ARGS="-gvisor"
 mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
 sudo install cron/cleanup_and_reboot_Linux.sh /etc/cron.hourly/cleanup_and_reboot || echo "FAILED TO INSTALL CLEANUP"
 
+sudo apt-get -y install qemu-system libvirt-clients libvirt-daemon-system
```
Comment (priyawadhwa) [inline hack/jenkins/linux_integration_tests_kvm.sh:39]: do we need `sudo apt-get update` before this?

Comment (sharifelgamal) [reply hack/jenkins/linux_integration_tests_kvm.sh:40]: almost certainly. fixed.

File: hack/jenkins/linux_integration_tests_kvm.sh [hunk 2]
```diff
@@ -36,4 +36,9 @@ EXTRA_TEST_ARGS="-gvisor"
 mkdir -p cron && gsutil -qm rsync "gs://minikube-builds/${MINIKUBE_LOCATION}/cron" cron || echo "FAILED TO GET CRON FILES"
 sudo install cron/cleanup_and_reboot_Linux.sh /etc/cron.hourly/cleanup_and_reboot || echo "FAILED TO INSTALL CLEANUP"
 
+sudo apt-get update
+sudo apt-get -y install qemu-system libvirt-clients libvirt-daemon-system
+sudo adduser jenkins libvirt
```
Comment (medyagh) [inline hack/jenkins/linux_integration_tests_kvm.sh:41]: we probabl need || true if the user was allready added, I think it will give error

**Review by medyagh — CHANGES_REQUESTED** [review]

File: hack/jenkins/linux_integration_tests_kvm.sh (hunk 2 above)
Comment (sharifelgamal) [reply hack/jenkins/linux_integration_tests_kvm.sh:41]: fixed

**Review by priyawadhwa — APPROVED** [review]

**Review by medyagh — APPROVED** [review]

**Pull request merged**